pinot-core/src/main/java/org/apache/pinot/core/segment/index/loader/defaultcolumn/BaseDefaultColumnHandler.java

```diff
@@ -164,16 +164,16 @@ public void updateDefaultColumns(IndexLoadingConfig indexLoadingConfig)
    *
    * @return Action Map for each column.
    */
-  private Map<String, DefaultColumnAction> computeDefaultColumnActionMap() {
+  static Map<String, DefaultColumnAction> computeDefaultColumnActionMap(Schema schema, SegmentMetadataImpl segmentMetadata) {
```

Comment:
Can we revert this as well? Seems only used in the test

Reply:
reverted most changes. kept the unit test